timeless/auth/auth.py

```diff
@@ -0,0 +1,16 @@
+from flask import session
+from timeless.employees.models import Employee
+
+
+def login(username="", password=""):
+    """Login user"""
+    user = Employee.query.filter_by(username=username).first()
+    error = None
+    if user is None:
+        error = "Incorrect username."
+    elif not user.validate_password(password):
+        error = "Incorrect password."
```

Comment:
@Vatavuk The backend should return only error codes, not hardcoded messages. The codes should then be translated into (mapped to) Strings on the UI, so we can easily have internationalization or change error messages without having to rebuild + redeploy the backend.

Reply:
@amihaiemil I agree, but I believe that this should be done in the upper layer, in the view, not here.

Reply:
@Vatavuk I think it would be fine to do return it here, but that's just my opinion. Yes, it's fine to leave a puzzle :)

Reply:
@amihaiemil I've misunderstood you, yes messages should be change to codes in this class. I've put a puzzle there.